lib/components/Field/NumberField.tsx

```diff
@@ -119,10 +119,11 @@ export const NumberField: React.StatelessComponent<NumberFieldProps> = (props: N
                 max={props.max}
                 required={props.required}
                 attr={numberAttr}
+                ref={ref}
```

Comment:
I'm not sure if this collide with the original design. @vishwam do you have more context on this?